docusaurus/docs/render-prop.md

```diff
@@ -30,6 +30,10 @@ import { ReduxFetch } from 'react-redux-fetch'; // ES6
 
   Called every time one of the configured resources transitions from the pending to the fulfilled state.
 
+- `onFulfilAll`: **() => void [optional]**
+
+  Called when all resources transitioned from the pending to the fulfilled state.
```

Comment:
Make it more clear that only the resources for the given config are checked and not all resources in the application. E.g.: when all resources **for this component** transitioned ...